packages/gestalt/src/utils/maybeTooltip.js

```diff
@@ -0,0 +1,35 @@
+// @flow strict
```

Comment:
This file represents a component, so should use PascalCase

Reply:
fixed